plugins/masters-tournament/masters_data.py

```
@@ -17,26 +17,39 @@
-    LEADERBOARD_URL = "https://site.api.espn.com/apis/site/v2/sports/golf/pga/leaderboard"
-    SCHEDULE_URL = "https://site.api.espn.com/apis/site/v2/sports/golf/pga/schedule"
-    NEWS_URL = "https://site.api.espn.com/apis/site/v2/sports/golf/pga/news"
+    # NOTE: ESPN's legacy `/sports/golf/pga/leaderboard` path now 404s.
+    # The current working endpoint drops the `pga` segment.
+    LEADERBOARD_URL = "https://site.api.espn.com/apis/site/v2/sports/golf/leaderboard"
+    NEWS_URL = "https://site.api.espn.com/apis/site/v2/sports/golf/news"
+    # Athlete bio + overview live on site.web.api (common/v3), not site.api.
+    ATHLETE_URL = "https://site.web.api.espn.com/apis/common/v3/sports/golf/pga/athletes/{player_id}"
+    ATHLETE_OVERVIEW_URL = "https://site.web.api.espn.com/apis/common/v3/sports/golf/pga/athletes/{player_id}/overview"
+
+    HTTP_HEADERS = {"User-Agent": "LEDMatrix Masters Plugin/2.1"}
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)